pkgs/applications/misc/keystore-explorer/default.nix

```diff
@@ -1,13 +1,14 @@
-{ fetchzip, lib, stdenv, jdk, runtimeShell }:
+{ fetchzip, lib, stdenv, jdk, runtimeShell, glib, wrapGAppsHook }:
 
 stdenv.mkDerivation rec {
-  version = "5.4.4";
+  version = "5.5.1";
   pname = "keystore-explorer";
   src = fetchzip {
-    url = "https://github.com/kaikramer/keystore-explorer/releases/download/v${version}/kse-544.zip";
-    sha256 = "01kpa8g6p6vcqq9y70w5bm8jbw4kp55pbywj2zrhgjibrhgjqi0b";
+    url = "https://github.com/kaikramer/keystore-explorer/releases/download/v${version}/kse-551.zip";
+    sha256 = "2C/LkUUuef30PkN7HL0CtcNOjR5uNo9XaCiTatv5hgA=";
   };
 
+  nativeBuildInputs = [ glib wrapGAppsHook ];
```

Comment:
Why do we need glib here?

Reply:
This actually belongs in buildInputs, you are right. Without it, system file dialogs don't open and the program hangs.